src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1337,4 +1337,7 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="UnableToPurgeInstallDirectory" xml:space="preserve">
     <value>Cannot purge install directory, as it was not created by WinGet</value>
   </data>
+  <data name="ShowLabelDocumentations" xml:space="preserve">
+    <value>Documentations:</value>

```

Comment:
"Documentations" feels weird to me. Maybe we can add a comment to be sure the loc team translates it properly?

Reply:
I added a comment to clarify the meaning of "Documentations"

Reply:
Changed to `Documentation`